R/supervised-model-development.R

```diff
@@ -26,6 +26,7 @@ SupervisedModelDevelopment <- R6Class("SupervisedModelDevelopment",
     grainTest = NA,
     prevalence = NA,
     factorLevels = NA,
+    imputeVals = NA,
```

Comment:
For now, let's leave the pipeline as it is and let the user have the function if they want to impute on their own. In the future, we can add that to the advanced class.